docs/root/intro/deprecated.rst

```diff
@@ -21,6 +21,9 @@ Version 1.11.0 (Pending)
 * Use of :ref:`runtime <envoy_api_field_config.bootstrap.v2.Bootstrap.runtime>` in :ref:`Bootstrap
   <envoy_api_msg_config.bootstrap.v2.Bootstrap>`. Use :ref:`layered_runtime
   <envoy_api_field_config.bootstrap.v2.Bootstrap.layered_runtime>` instead.
+* Specifying "deprecated_v1: true" in HTTP and network filter configuration to allow loading JSON
```

Comment:
Do you think it's worth mentioning the runtime control either in the relnotes, the log message, or both?

Reply:
Yeah good call let me do that.